t/rosa-db-create/00-basic.t

```diff
@@ -22,6 +22,8 @@
 # svn-post-commit", which is tested quite thoroughly in its own test suite.
 #-------------------------------------------------------------------------------
 . $(dirname $0)/test_header
+skip_all # Skipped pending fix
```

Comment:
Why is ``skip_all`` called twice for many of the skipped tests?

Reply:
Probably a `Tim: Cannot use SED competently error`